substrate-node/pallets/pallet-tft-price/src/lib.rs

```diff
@@ -130,8 +136,26 @@ pub mod pallet {
     #[pallet::getter(fn allowed_origin)]
     pub type AllowedOrigin<T: Config> = StorageValue<_, T::AccountId, OptionQuery>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn min_tft_price)]
+    pub type MinTftPrice<T> = StorageValue<_, u32, ValueQuery>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn max_tft_price)]
+    pub type MaxTftPrice<T> = StorageValue<_, u32, ValueQuery>;
+
     #[pallet::call]
     impl<T: Config> Pallet<T> {
+        #[pallet::weight(100_000_000 + T::DbWeight::get().writes(1))]
```

Comment:
please revert order of extrinsics:

- set_prices
- set_allowed_origin
- newer ones

Reply:
done